### DISCUSSION TITLE
Getting an `ambiguous argument` fatal error when passing HEAD^1 to the Task Filtering API

### DISCUSSION THREAD

**Discussion by nabilk2** [vercel/turborepo#5522] [Help] [answered]
I created a new monorepo following this article: https://heygrady.com/posts/2022-09-01-creating-a-yarn-monorepo-in-2022/

Once I followed the set up, the pre-commit hook runs with the following command:

 `yarn turbo run --concurrency=1 --filter=[HEAD^1] precommit`

Whenever I run this I get the following error: 
```
Turbo error: failed to resolve packages to run: git error: fatal: ambiguous argument 'HEAD^1...HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Does anyone know how I can get over this error? I do want to mention that this is the initial commit for the branch, so I guess it would make sense why I get the error but this would also be not ideal for future new branches.

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this, please open an issue. 🙏
